pyproject.toml

```diff
@@ -49,7 +49,9 @@ markdown-include = "^0.5.1"
 black = { version = "^22.6.0", markers = "implementation_name != 'pypy'" }
 isort = "^5.10.1"
 pre-commit = "^2.20.0"
-types-backports = "^0.1.3"
```

Comment:
Don't we need that for `zoneinfo`?

Reply:
No. `backports.zoneinfo` includes stub files. For some reason the type check isn't working on Windows and seems to be a [mypy bug](https://github.com/python/mypy/issues/14057#issuecomment-1334528626).